modules/serverpod_auth/serverpod_auth_sms_flutter/pubspec.yaml

```diff
@@ -0,0 +1,63 @@
+# This file is generated. Do not modify, instead edit the files in the templates/pubspecs directory.
```

Comment:
Suggestion: This pubspec needs to be reflected in the templates folder.

Please check the other flutter pubspecs in the template folder.